classes/oc/i18n.php

```diff
@@ -76,7 +76,7 @@ public static function initialize($locale = NULL, $charset = 'utf-8', $domain =
          * and force using gettext
          */
         if(defined('LC_ALL'))
-            $locale_res = setlocale(LC_ALL, self::$locale);
+            $locale_res = setlocale(LC_ALL, self::$locale.'.'.self::$charset);
```

Comment:
Let me.investigate as soon as in the office.

On Tue, Sep 1, 2015, 05:11 Oliver <notifications@github.com> wrote:

> In classes/oc/i18n.php
> <https://github.com/open-classifieds/common/pull/111#discussion_r38382235>
> :
>
> > @@ -76,7 +76,7 @@ public static function initialize($locale = NULL, $charset = 'utf-8', $domain =
> >           * and force using gettext
> >           */
> >          if(defined('LC_ALL'))
> > -            $locale_res = setlocale(LC_ALL, self::$locale);
> > +            $locale_res = setlocale(LC_ALL, self::$locale.'.'.self::$charset);
>
> I think this works fine on Linux but can’t be entirely sure about other
> OS's.
>
> If not I can’t see how to get localeconv() information with the right
> charset.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/open-classifieds/common/pull/111/files#r38382235>.
>